Dockerfile

```diff
@@ -1,29 +1,28 @@
 # Copyright 2024 OpenVPN Inc <sales@openvpn.net>
 # SPDX-License-Identifier: Apache-2.0
 #
-FROM --platform=$TARGETPLATFORM ubuntu:22.04
-
-ARG TARGETPLATFORM
-ARG VERSION
-ARG DEBIAN_FRONTEND="noninteractive"
-LABEL maintainer="pkg@openvpn.net"
-
-RUN apt-get update && \
-    apt-get install -y \
-    ca-certificates \
-    wget \
-    gnupg \
-    net-tools \
-    iptables \
-    systemctl
-
-RUN wget https://as-repository.openvpn.net/as-repo-public.asc -qO /etc/apt/keyrings/as-repository.asc && \
-    echo "deb [arch=${TARGETPLATFORM#linux/} signed-by=/etc/apt/keyrings/as-repository.asc] http://as-repository.openvpn.net/as/debian jammy main">/etc/apt/sources.list.d/openvpn-as-repo.list && \
-    apt update && apt -y install openvpn-as=$VERSION && \
-    apt clean
-
-RUN mkdir -p /openvpn /ovpn/tmp /ovpn/sock && \
-    sed -i 's#~/tmp#/ovpn/tmp#g;s#~/sock#/ovpn/sock#g' /usr/local/openvpn_as/etc/as_templ.conf
+FROM ubuntu:22.04
+
+LABEL org.opencontainers.image.authors="pkg@openvpn.net" \
+      org.opencontainers.image.description="docker openvpn-as"
+
+ARG TARGETPLATFORM \
+    VERSION \
+    DEBIAN_FRONTEND="noninteractive"
+
+RUN apt-get update \
```

Comment:
I understand that putting 3 `RUN` into one reduces layers but what's the value here?
There are so few of them.
I would say having 3 layers here have more benefits, for example save some time during rebuilding

Reply:
As you wish)